core/src/main/java/com/graphhopper/util/Instruction.java

```diff
@@ -238,6 +238,20 @@ void checkOne() {
             throw new IllegalStateException("Instruction must contain at least one point " + toString());
     }
 
+    /**
+     * This method returns the length of an Instruction. The length of an instruction is defined by [the
+     * index of the first point of the next instruction] - [the index of the first point of this instruction].
+     * <p>
+     * In general this will just resolve ot the size of the PointList. This is not true for {@link ViaInstruction} and
```

Comment:
Would you write? 'In general this will just resolve ot the size of the PointList except for ...' because the 'last-first' definition still holds for ViaInstructions and I first thought 'this is not true' was referring to the definition